buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -5,6 +5,7 @@ import org.gradle.kotlin.dsl.exclude
 import org.gradle.plugin.use.PluginDependenciesSpec
 
 object Versions {
+    const val Arrow = "1.0.1"
```

Comment:
The latest Arrow documentation is very lacking :( It was fun learning about all this jazz though